app/scripts/modules/core/src/pipeline/executions/execution/Execution.tsx

```diff
@@ -13,11 +13,13 @@ import { IExecutionViewState, IPipelineGraphNode } from 'core/pipeline/config/gr
 import { OrchestratedItemRunningTime } from './OrchestratedItemRunningTime';
 import { SETTINGS } from 'core/config/settings';
 import { AccountTag } from 'core/account';
-import { NgReact, ReactInjector } from 'core/reactShims';
+import { ReactInjector } from 'core/reactShims';
 import { duration, timestamp } from 'core/utils/timeFormatters';
 import { ISortFilter } from 'core/filterModel';
 import { ExecutionState } from 'core/state';
 
+import { CopyToClipboard } from '@spinnaker/core';
```

Comment:
```suggestion
import { CopyToClipboard } from 'core/utils';
```
The import rules are unfortunately confusing.   When inside the `core` module, we import from the `core/[section]` alias.  When inside a non-core module (such as `kubernetes` or `amazon`) we import from `@spinnaker/core`